example/template.dart

```diff
@@ -21,7 +21,7 @@ class TemplateBenchmark extends BenchmarkBase {
   @override
   void setup() {}
 
-  // Not measures teardown code executed after the benchark runs.
+  // Not measures teardown code executed after the benchmark runs.
```

Comment:
This sentence doesn't read well even w/ the spelling fix :) Should this be 'Note that the time spent in teardown doesn't contribute to the benchmark time.' ?